docs/content/docs/build/releases-and-apitags.md

```diff
@@ -37,10 +37,18 @@ There are no guarantees for API breakages or Fluid document compatibility among
 
 ## API Tags
 
-There are no API guarantees for packages in the @fluid-experimental and @fluid-internal namespaces. @fluid-experimental APIs are for you to try and provide feedback on and @fluid-internal APIs are only meant for internal consumption by the framework and should not be used.
-For packages that are part of the **@fluidframework** namespace, we use API tagging to communicate the stability and guarantees associated with those APIs.
-
--   **Public APIs** (`@public`) - These are officially released and supported by us. Any breaking changes to these APIs will first be deprecated before being removed in a major release. We guarantee not to break them in minor and patch releases (respecting the semver guarantees)
--   **Beta APIs** (`@beta`) - These APIs are on the path to being officially supported but can still change in form before becoming a Public API. They are meant as a preview, for developers to experiment with and provide feedback. These APIs can change even in minor releases. As a result, prod use is discouraged.
--   **Legacy APIs** (`@legacy`) - These are early Fluid APIs used by early adopters of Fluid Framework. They are on a path to deprecation in a future release and we strongly encourage all new applications to avoid these APIs. Existing users of SharedMap DDS will need to use the legacy API until we provide a migration path to SharedTree in a future release.
--   **Internal APIs** (`@internal`) - Internal Fluid APIs, should not be used by applications.
+There are no API stability guarantees for packages in the `@fluid-experimental` and `@fluid-internal` scopes.
+`@fluid-experimental` APIs are for developers to try experimental features and provide feedback. It's possible that these APIs can get completely scrapped or drastically changed in a minor release based on developer feedback.
+`@fluid-internal` APIs are only meant for internal consumption by the framework and should not be used.
+
+For packages that are part of the **@fluidframework** scope, we use import paths to communicate the stability and guarantees associated with those APIs.
```

Comment:
Is this meant for 1p or general? General we only care about `fluid-framework` (which is not mentioned). For [1p] using `@fluidframework` we would want to note that /legacy should be treated as unstable (can change in minor) until we note otherwise.

(I don't like making it complicated, but it is a bit :( )

Reply:
This is for general external guidance, I think for 1P we should have publish that somewhere internally to not complicate the external messaging. But I'll update the pre-condition to include the fluid-framework package.